admin/views/tool-file-editor.php

```diff
@@ -67,7 +67,11 @@
 	echo '<div id="message" style="width:94%;" class="updated fade"><p>', esc_html( $msg ), '</p></div>';
 }
 
-$action_url = network_admin_url( 'admin.php?page=wpseo_tools&tool=file-editor' ); // auto-falls back on admin_url for non-multisite
+if ( is_multisite() ) {
```

Comment:
This makes inline comments and `network_admin_url()` in `else` redundant, it's either multisite or not.